packages/backend/src/api-impl.ts

```diff
@@ -34,6 +38,54 @@ export class BootcApiImpl implements BootcApi {
     return buildDiskImage(build, this.history);
   }
 
+  async deleteBuild(build: BootcBuildInfo): Promise<void> {
```

Comment:
IMHO it feels like overkill to have and expose two separate functions for deleting one vs many. I'd prefer to just keep one as we have in PD. e.g. easy enough to do `deleteBuilds([one])` when necessary and have a single delete implementation.